assets/finamp.desktop.m4

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
+Name=Finamp
+GenericName=Music Player
+Icon=finamp
```

Comment:
does that works already, or do maintainers need to adapt that to their package?

Reply:
As long as the icons are also installed according to the [XDG icon theme spec][spec], yes. 

[spec]: https://specifications.freedesktop.org/icon-theme-spec/icon-theme-spec-latest.html